src/util-prefilter.h

```diff
@@ -61,25 +61,24 @@ int PrefilterAddSidsResize(PrefilterRuleStore *pmq, uint32_t new_size);
 static inline void PrefilterAddSids(
         PrefilterRuleStore *pmq, const SigIntId *sids, uint32_t sids_size)
 {
-    if (sids_size == 0)
-        return;
-
-    uint32_t new_size = pmq->rule_id_array_cnt + sids_size;
-    if (new_size > pmq->rule_id_array_size) {
-        if (PrefilterAddSidsResize(pmq, new_size) == 0) {
-            // Failed to allocate larger memory for all the SIDS, but
-            // keep as many as we can.
-            sids_size = pmq->rule_id_array_size - pmq->rule_id_array_cnt;
+    if (sids_size > 0) {
```

Comment:
Not following why is an extra indentation preferable over return if base condition doesn't match.

Reply:
The logic I think is that it assumes the first branch (so sids_size == 0 leading to return) to be the most likely.